src/coreclr/CMakeLists.txt

```
@@ -126,6 +126,10 @@ if(FEATURE_STANDALONE_GC)
   endif (CLR_CMAKE_BUILD_SUBSET_RUNTIME)
 endif(FEATURE_STANDALONE_GC)
 
+if(NOT CLR_CROSS_COMPONENTS_BUILD)
+  add_subdirectory(${CLR_REPO_ROOT_DIR}/src/installer/corehost/cli/apphost/static Corehost.Static)
```

nit: after the rebase, this can be simplified to:
```suggestion
  add_subdirectory(${CLR_ENG_NATIVE_DIR}/corehost/apphost/static Corehost.Static)
```